src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -218,22 +227,25 @@ def get_event_stats_metrics(_, user_query, params, rollup, zerofill_results, __)
                 int(request.GET.get("topEvents", DEFAULT_TOP_EVENTS_LIMIT)),
                 MAX_TOP_EVENTS_LIMIT,
             )
+
             # Fetch transactions names with the highest event count
-            top_events = get_top_events(
+            global top_trending_transactions
```

Comment:
I think you want `nonlocal` here instead of `global`

Reply:
haha yes!